.travis.yml

```diff
@@ -1,14 +1,14 @@
 sudo: false
 language: python
 python:
-  - "2.7"
+  - "2.7.10"
 addons:
   postgresql: "9.1"
 
 services:
   - postgresql
 install:
-  - pip install -r requirements.txt --use-mirrors
+  - pip install -r requirements.txt
```

Comment:
Yeah, these changes can be removed. I just added them because I didn't have a 2.7.9 environment for testing. The build works without these also.